rented-properties/src/main/java/org/egov/cpt/service/RentDemandGenerationService.java

```diff
@@ -60,8 +61,8 @@ public RentDemandGenerationService(PropertyRepository propertyRepository, Produc
 		this.demandNotificationService = demandNotificationService;
 	}
 
-	public void createDemand(RentDemandCriteria demandCriteria) {
-
+	public AtomicInteger createDemand(RentDemandCriteria demandCriteria) {
```

Comment:
I don't know if we should use an atomic integer. We are only going to use this here in this function. Also, each request runs on a different thread.

Reply:
We can not use integer counter within for loop so I used AtomicInteger

Reply:
counter is effectively final inside the lamda expression. The other way to do this is to use map instead of forEach and reduce them with `collect(Collectors.summingInt(Integer::intValue));` 
Merging this anyway.